modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/GatlingCodegen.java

```diff
@@ -11,7 +11,7 @@
 import java.util.*;
 import java.io.File;
 
-public class GatlingCodegen extends DefaultCodegen implements CodegenConfig {
+public class GatlingCodegen extends AbstractScalaCodegen implements CodegenConfig {
```

Comment:
The idea behind AbstractScalaCodegen is that a lot of code is factorized in the class. So you should be able to remove a lot of code from GatlingCodegen.

Reply:
I will look it over later today and remove whatever I can.  Im assuming this will help with my java.util.List getting imported as well?

Reply:
You should be able to use common importMapping, typeMapping, reserved words and pre/post-processing methods.

Reply:
Yeah, i used the importMapping and removed List as you can see but sometimes it ends up getting imported into my model template.

Reply:
It did fix the list issue.  Thank you.

Reply:
The only other issue that I have is that scala has a reservedWord as type, but i know many apis will have the word type in them. So for the JSON string i build out, it contians the _type as a field.  

Im assuming this is where I could use jackson but even if i do that, when i call {{name}} from the {{vars}} its going to show up as _type anyways.  It would be really nice if the reservedWords gave an additional field to use as variables instead of changing the name itself.

Reply:
You can escape reserved words directly.

See how I did this in the Kotlin client recently:
https://github.com/swagger-api/swagger-codegen/pull/6874/files

I'd be fine with Scala escaping similarly.